base/set.jl

```diff
@@ -191,6 +191,24 @@ function unique(f::Callable, C)
 end
 
 """
+	unique!(itr)
```

Comment:
Removing elements is generally not possible on iterators, so better replace `itr` with `c` (for collection).